testsuite/gna/issue2005/testsuite.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/sh
+
+. ../../testenv.sh
+
+set -x
+
+if c_compiler_is_available; then
+	lib=$("$GHDL" --libghdl-name)
+	lib=${lib#lib} # strip "lib" prefix
+	lib=${lib%.so} # strip .so suffix
+	libdir="$(dirname "$("$GHDL" --libghdl-library-path)")"
+
+
+	$CC -I"$("$GHDL" --libghdl-include-dir)" test.c "$("$GHDL" --libghdl-library-path)"
+	$CC -I"$("$GHDL" --libghdl-include-dir)" -L$libdir test.c -l$lib
```

Comment:
I'm testing both ways now, straight up library path and searching --libghdl-name in the --libghdl-library-path dir.